examples/rondpoint/tests/bindings/test_rondpoint.swift

```diff
@@ -1,10 +1,10 @@
 import rondpoint
 
 let dico = Dictionnaire(un: .deux, deux: false)
-let copyDico = copieDictionnaire(d: dico)
+let copyDico = try! copieDictionnaire(d: dico)
```

Comment:
Oh wow, do we now have to wrap every call to swift functions in a `try!` because of the error handling?

Reply:
Actually looks like there was some discussion of this with @linacambridge already below, so don't feel like you need to respond to this directly :-)

Reply:
I'll have this among a couple 'tiny' cleanups in a follow-up PR, also feel free to add any more opinions/reviews on the PR, and I can address them too! Wanted to merge the big chunk of this so that other PRs don't need to wait and we can iterate on smaller PRs improving the error handling.